spec/requests/api/v1/articles/show_spec.rb

```diff
@@ -1,5 +1,10 @@
 RSpec.describe "GET /api/v1/articles", type: :request do
-  let!(:article) { create(:article) }
+  let!(:journalist) { create(:user, role: "journalist") }
```

Comment:
Why are you creating a journalist?

Reply:
We have an assoication that an article need a journalist. Then we need a journalist to create an article?